from-dump/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quetzacoalt/docker-internal-images:1.7
```

Comment:
It is based on `prestashop/prestashop:1.7` (https://github.com/PrestaShop/docker-internal-images/blob/master/1.7/Dockerfile#L1)

At the beginning, this image was private, we needed my personal account to make this possible without a premium account.

Although I know we drop the whole PrestaShop folder to put another one, I just reuse the existing layers instead of copy-pasting dockerfiles.